server/services/v1/database/query_runner.go

```diff
@@ -646,13 +661,37 @@ func (runner *StreamingQueryRunner) iterate(coll *schema.DefaultCollection, iter
 			return row.Key, err
 		}
 
+		if isAcceptApplicationJSON {
+			buffResponse = append(buffResponse, &api.ReadResponse{
```

Comment:
api.ReadResponse createion looks mostly identical in both path move it out.
set ResumeToken separately.